Channel islands state to avoid issues

TheBrit79 opened the pull request.

TheBrit79 committed:
Channel islands state to avoid issues

marijn211 reviewed (changes requested):
Please wait with accepting so we don't break all saves again
(Ask someone else if you want to dismiss the review, Im not working on the mod much atm)

hammmurabi commented:
> Please wait with accepting so we don't break all saves again
> (Ask someone else if you want to dismiss the review, Im not working on the mod much atm)

something needs to be done about them since right now UK can do land trade with all of europe and will never do dday if they aren't occupied

They either need to be made a new state, have the crossing removed, or be removed entirely

One of the 3 and it needs to be in the next update or it's just impossible to convoy raid UK and dday will never happen until august

SwedishNorwegian commented:
Per Spicy's instructions, this pull request is not to be merged until the next Map Changing pass which is August.

TheBrit79 committed:
Merge branch 'Beta-Edits' into Maps-August

TheBrit79 committed:
Merge branch 'Beta-Edits' into Maps-August

TheBrit79 committed:
Malaysia

TheBrit79 committed:
More Malaysia

TheBrit79 committed:
Bolivia state and South American airzones

hammmurabi committed:
Revert "Reversion"

This reverts commit 791e5f2ec73e9437d89421c9bfbad2516b8d19a3.

hammmurabi committed:
sudetenland south

hammmurabi committed:
final south sudetenland fix

TheBrit79 committed:
V

TheBrit79 committed:
Merge branch 'Maps-August' of https://github.com/deliciousmods/1956_beta into Maps-August

hammmurabi committed:
Update buildings.txt

TheBrit79 committed:
Update 14-Brittany.txt

TheBrit79 committed:
Merge branch 'Beta-Edits' into Maps-August

TheBrit79 committed:
Maps

TheBrit79 committed:
Provinces

TheBrit79 committed:
Update rivers.bmp

hammmurabi committed:
Update rivers.bmp

TheBrit79 committed:
Luxembourg

SwedishNorwegian committed:
Small balance adjustment

Tweaking some things here and there

TheBrit79 committed:
Revert "Update rivers.bmp"

This reverts commit 025fd030c29b5104490cabf36c414b21a85d33aa.

hammmurabi committed:
austria

TheBrit79 committed:
Central American provinces

TheBrit79 committed:
Update victory_points_l_english.yml

TheBrit79 committed:
Merge branch 'Beta-Edits' into Maps-August

TheBrit79 committed:
Random VPs

TheBrit79 committed:
Operation make Mexico slightly less cancerous

marijn211 committed:
Merge branch 'Beta-Edits' into Maps-August

TheBrit79 committed:
Belize that

TheBrit79 committed:
Merge branch 'Maps-August' of https://github.com/deliciousmods/1956_beta into Maps-August

TheBrit79 committed:
Merge branch 'Beta-Edits' into Maps-August

TheBrit79 committed:
Rawr

TheBrit79 committed:
Pacific islands boatload

TheBrit79 committed:
British railways and Scottish rivers

TheBrit79 committed:
Work damn you

SwedishNorwegian committed:
.

SwedishNorwegian committed:
Merge branch 'Maps-August' of https://github.com/deliciousmods/1956_beta into Maps-August

TheBrit79 committed:
Loc

TheBrit79 committed:
Merge branch 'Maps-August' of https://github.com/deliciousmods/1956_beta into Maps-August

marijn211 committed:
Moved Memel back to the coast

TheBrit79 committed:
Merge branch 'Beta-Edits' into Maps-August

hammmurabi committed:
Merge branch 'Beta-Edits' into Maps-August

hammmurabi committed:
add hebron province to fix jerusalem frontline bug

hammmurabi committed:
revert

TheBrit79 committed:
Minor content addition

TheBrit79 committed:
Merge branch 'Maps-August' of https://github.com/deliciousmods/1956_beta into Maps-August

hammmurabi committed:
hebron province to fix jerusalem frontline bug

hammmurabi committed:
f

hammmurabi committed:
Update r56_state_rename_scripted_localization.txt

hammmurabi committed:
Update state_names_l_english.yml

TheBrit79 committed:
Merge branch 'Beta-Edits' into Maps-August

TheBrit79 committed:
Random stuff

TheBrit79 committed:
Tuvalu

TheBrit79 committed:
VP

TheBrit79 committed:
Africa

TheBrit79 committed:
Burgenland

TheBrit79 committed:
Update railways.txt

TheBrit79 committed:
Update adjacencies.csv

TheBrit79 committed:
Greenland

SwedishNorwegian committed:
Change to Vinnytsia and MP Tweak in Tank Chassis

SwedishNorwegian committed:
Merge branch 'Maps-August' of https://github.com/deliciousmods/1956_beta into Maps-August

TheBrit79 committed:
Guyana

TheBrit79 committed:
Update provinces.bmp

TheBrit79 committed:
Merge branch 'Maps-August' of https://github.com/deliciousmods/1956_beta into Maps-August

LemuelSora committed:
Hun Core change

Instead of Hungary starting with a claim on Uzhhorod they start with a claim and gain the core via Reaffirm Territorial Claims.

TheBrit79 committed:
VP

TheBrit7 committed:
Merge branch 'Beta-Edits' into Maps-August

TheBrit7 committed:
Various bugfixes and portraits

TheBrit7 committed:
Chatham islands

TheBrit7 committed:
Update definition.csv

TheBrit7 committed:
Still got some work to do

TheBrit7 committed:
Still work to do

hammmurabi committed:
fix river error

TheBrit7 committed:
Work

TheBrit7 committed:
Merge branch 'Maps-August' of https://github.com/deliciousmods/1956_beta into Maps-August

hammmurabi commented on map/provinces.bmp:
map is expanded to 5632x2112 (for no reason, nothing is added in the additional southern ocean added) which is not divisible by 128 and over the map's size limit causing a giant line to form at the top of the map

hammmurabi reviewed (changes requested):
map is expanded southwards from the correct size of 5632x2048 to 5632x2112 (for no reason, nothing is added in the new area) which is not divisible by 256 and over the map's size limit, causing potential crashes and a giant line to appear at the top of the map + heightmap issues

TheBrit7 committed:
Revert "Work"

This reverts commit b47bfe2b5edf79ae3b8a2f1fe7c3d70df5785054.

TheBrit7 committed:
Revert "Still work to do"

This reverts commit 8bf7e781dbd730dd65a7cf7ec64e3b59aa0fb533.

TheBrit7 committed:
Anti-Work

TheBrit7 committed:
Revert "Anti-Work"

This reverts commit a52e7a2289ab0203cce1ad9565753dedb6dc6328.

TheBrit7 committed:
Help
